cli/functionary/environment.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+import click
+import requests
+
+from .config import get_config_value, save_config_value
+
+
+def get_environment_list():
+    """
+    Helper function to get the environment list from host
+
+    Args:
+        None
+
+    Returns:
+        None
+
+    Raises:
+        ClickException if bad get request
+    """
+    token = get_config_value("token")
+    teams_url = get_config_value("host") + "/api/v1/teams"
+    header = {"Authorization": f"Token {token}"}
+    response = requests.get(teams_url, headers=header)
+
+    if response.ok:
+        data = json.loads(response.text).get("results")
+
+        env_list = []
+        for team in data:
+            for env_set in team.get("environments"):
+                env_list.append(env_set)
+        return env_list
+    else:
+        raise click.ClickException(
+            f"Failed to get environment list: {response.status_code}\n"
+            f"Response: {response.text}"
+        )
+
+
+@click.group("environment")
+@click.pass_context
+def environment_cmd(ctx):
+    pass
+
+
+@environment_cmd.command()
+@click.pass_context
+def set(ctx):
+    """
+    Command to set the environment id based on user input
```

Comment:
You'll want this summary line to read as something that will be useful to someone running the `functionary environment` command, since the help text will be populated from here.  Something simple like:

> Set the active environment